app/_src/kubernetes-ingress-controller/guides/high-availability/health-checks.md

```diff
@@ -4,3 +4,335 @@ type: how-to
 purpose: |
   How to enable passive and active health checks for upstream services
 ---
+
+Setup active and passive health checking using the {{site.kic_product_name}}. This configuration allows
+Kong to automatically short-circuit requests to specific Pods that are mis-behaving in your Kubernetes Cluster.
+
+When you setup a passive health check for a service that runs in a cluster and if the Pod that runs the service reports an error, Kong returns a 503, indicating that the service is unavailable. Kong does not proxy anymore requests.
+There are a few options that you can consider:
+
+- Delete the current Pod; Kong then sends proxy requests to the new Pod that comes in its place.
+- Scale the deployment; Kong then sends proxy requests to the new Pods and leave the short-circuited Pod out of the loop.
+- Manually change the Pod health status in Kong using Kong's [Admin API](/gateway/latest/admin-api/#set-target-as-healthy).
+
+These options highlight the fact that once a circuit is opened because of errors, there is no way for Kong to close the circuit again. Manual intervention is necessary so that a Pod can again handle requests.
+You can use active health-check, where each instance of Kong actively probes Pods to check if they are healthy.
+
+
+{% include_cached /md/kic/prerequisites.md kong_version=page.kong_version disable_gateway_api=false %}
+
+## Create a Kubernetes service
+
+Create a Kubernetes service setup a [httpbin](https://httpbin.org)
+service in the cluster and proxy it.
+
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{site.data.kong_latest_KIC.version}}/deploy/manifests/httpbin.yaml
+```
+The results should look like this:
+
+```text
+service/httpbin created
+deployment.apps/httpbin created
+```
+
+## Make sure service is accessible outside cluster
+
+1. Expose the service outside the Kubernetes cluster.
+
+{% navtabs api %}
```

Comment:
I couldn't make the navtabs work as expected when the content is indented. I'm not sure if it's possible to use it like that. Because I unindented the content, enumerating is broken now.

Reply:
I fixed this.